Database/funcs/UpdateReview.sql

```diff
@@ -0,0 +1,45 @@
+-- ================================================
+-- Template generated from Template Explorer using:
+-- Create Procedure (New Menu).SQL
+--
+-- Use the Specify Values for Template Parameters 
+-- command (Ctrl-Shift-M) to fill in the parameter 
+-- values below.
+--
+-- This block of comments will not be included in
+-- the definition of the procedure.
+-- ================================================
+SET ANSI_NULLS ON
+GO
+SET QUOTED_IDENTIFIER ON
+GO
+-- =============================================
+-- Author:		<Author,,Name>
+-- Create date: <Create Date,,>
+-- Description:	<Description,,>
+-- =============================================
+CREATE PROCEDURE [dbo].[UpdateReview]
+	-- Add the parameters for the stored procedure here
+	-- Add the parameters for the stored procedure here
+@ReviewID int,
+@UserID  int,
+@Hotel int,
+@Rating int,
+@Review nvarchar(MAX),
+@Date int

```

Comment:
Is date should be "int"? 'Cause in [https://camo.githubusercontent.com/e780e0495d065160464be133b87375fd6ec724e2b30a2c2a01feb61cc97db936/68747470733a2f2f692e6962622e636f2f384b6e334a66422f426f6f6b696e672e706e67](url) We use "date"

Reply:
We have decided that @Date should be of type date